pkg/instrumentation/defaultinstrumentation.go

```diff
@@ -14,10 +14,11 @@ import (
 )
 
 const (
-	defaultAPIVersion                      = "cloudwatch.aws.amazon.com/v1alpha1"
-	defaultInstrumenation                  = "java-instrumentation"
-	defaultNamespace                       = "default"
-	defaultKind                            = "Instrumentation"
+	defaultAPIVersion     = "cloudwatch.aws.amazon.com/v1alpha1"
```

Comment:
I personally find it hard to navigate these consts back & forth between the usages below and the values here when trying to see what the instrumentation CR actually looks like.
If these consts are not used in more than one place, I find inlining them to be more readable.. but might just be me.

Reply:
yeah I was debating on that too -- the only thing is that we do re-use the same env vars for each lang..maybe having an env map would be better